app/src/components/common/button_disconnect_wallet/index.tsx

```diff
@@ -1,47 +1,14 @@
 import React from 'react'
 import styled, { css } from 'styled-components'
```

Comment:
```suggestion
import styled from 'styled-components'
```
You should delete `{ css }` if it's not used.